charms/worker/k8s/lib/charms/k8s/v0/k8sd_api_manager.py

```diff
@@ -290,6 +290,24 @@ class UserFacingClusterConfig(BaseModel):
     metrics_server: MetricsServerConfig = Field(None, alias="metrics-server")
 
 
+class UserFacingDatastoreConfig(BaseModel):
```

Comment:
Thanks to @mateoflorido for this suggestion
```suggestion
class UserFacingDatastoreConfig(BaseModel, allow_population_by_field_name=True):
```

Reply:
Thanks, that fixed it!